src/scss/skin-modern/components/_uicontainer.scss

```diff
@@ -29,6 +29,12 @@
     }
   }
 
+  &.#{$prefix}-controls-shown {
+    .#{$prefix}-ui-hugeplaybacktogglebutton:focus {
+      box-shadow: inset -4px -3px 2px 9px rgba(27, 127, 204, 1) !important;
```

Comment:
- The `!important` here should not be necessary or? 🤔 
- If this is the bitmovin color you could use `rgba($color-highlight, .8);` If not, extract the color and use `rgba(color, alphy)`